src/Components/Aspire.Azure.AI.Inference/Aspire.Azure.AI.Inference.csproj

```diff
@@ -0,0 +1,30 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>$(DefaultTargetFramework)</TargetFramework>
+    <IsPackable>true</IsPackable>
+    <PackageTags>$(ComponentAzurePackageTags) ai</PackageTags>
+    <Description>A client for Azure AI Inference SDK that integrates with Aspire, including logging and telemetry.</Description>
+    <PackageIconFullPath>$(SharedDir)Azure_256x.png</PackageIconFullPath>
+    <NoWarn>$(NoWarn);SYSLIB1100;SYSLIB1101</NoWarn>
+    <!-- In preview until the public API is validated and the Microsoft.Extensions.AI integration is designed.. -->
+    <SuppressFinalPackageVersion>true</SuppressFinalPackageVersion>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="..\Common\AzureComponent.cs" Link="AzureComponent.cs" />
+    <Compile Include="..\Common\ConfigurationSchemaAttributes.cs" Link="ConfigurationSchemaAttributes.cs" />
```

Comment:
We should add the configuration schema attributes.

Reply:
Ah, must have missed doing that. I'll update the PR